### ISSUE TITLE
Review of 3rd Party Extensions

### ISSUE THREAD

**Issue by adamrusted** [simple-icons/simple-icons#9447]
As part of a wider review of the project, it'd be handy to confirm all 3rd party extensions we list on our `README` are actually up to date, and to a standard we're happy with as a project. Below is a list of all extensions - could anybody on the @simple-icons/maintainers team please verify that the icons included in the package are up-to-date, and that the functionality is up to scratch with what we'd recommend to users.

_(This should go without saying, but please don't review your own extension!)_

Please edit this post, tagging yourself next to the package you will check, or have already checked.
If any are found to be out of sync with the project, we should try to work with the creators of the package to get it back into sync, or sunset should they no longer be able to maintain it.

- [ ] [Angular module](https://github.com/avmaisak/ngx-simple-icons)
- [ ] [Blazor Nuget package](https://github.com/TimeWarpEngineering/timewarp-simple-icons)
- [ ] [Blender add-on](https://github.com/mondeja/simple-icons-blender)
- [ ] [DrawIO library](https://github.com/mondeja/simple-icons-drawio)
- [ ] [Drupal module](https://git.drupalcode.org/project/simple_icons)
- [x] [Figma plugin](https://github.com/LitoMore/simple-icons-figma) @LitoMore - verified by @adamrusted 
- [ ] [Flutter package](https://github.com/jlnrrg/simple_icons)
- [x] [Framer component](https://github.com/LitoMore/simple-icons-framer) @LitoMore - verified by @adamrusted 
- [ ] [Hexo plugin](https://github.com/nidbCN/hexo-simpleIcons)
- [ ] [Home Assistant plugin](https://github.com/vigonotion/hass-simpleicons)
- [ ] [Hugo module](https://github.com/foo-dogsquared/hugo-mod-simple-icons)
- [ ] [Java library](https://github.com/silentsoft/simpleicons4j)
- [ ] [Jetpack Compose library](https://github.com/DevSrSouza/compose-icons)
- [ ] [Kirby plugin](https://github.com/runxel/kirby3-simpleicons)
- [ ] [LaTeX package](https://github.com/ineshbose/simple-icons-latex)
- [ ] [Laravel package](https://github.com/ublabs/blade-simple-icons)
- [ ] [Leptos crate](https://github.com/Carlosted/leptos-icons)
- [ ] [Python package](https://github.com/sachinraja/simple-icons-py)
- [x] [React package](https://github.com/icons-pack/react-simple-icons) @LitoMore 
- [ ] [Ruby gem](https://github.com/the-pew-inc/simple-icons-rails)
- [ ] [Solid package](https://github.com/x64Bits/solid-icons)
- [ ] [Stream Deck icon pack](https://github.com/mackenly/simple-icons-stream-deck)
- [ ] [Svelte package](https://github.com/icons-pack/svelte-simple-icons)
- [ ] [Vue 3 package](https://github.com/wyatt-herkamp/vue3-simple-icons)
- [ ] [Vue package](https://github.com/mainvest/vue-simple-icons)
- [ ] [Wordpress plugin](https://plugins.trac.wordpress.org/browser/simple-icons/)

**Comment by mondeja** [comment]
I maintain the Blender addon and the Drawio library and are up to date, always synced with the latest version. The Leptos crate and icon have been added recently after my manual review and works fine.

Should the icons of the 3rd party extensions be synced with the versions of the project to mark them as properly reviewed?

**Comment by adamrusted** [comment]
I feel like in order to keep us out of hot water, they probably should.
Last thing we want is a brand coming after us because _X package still has our icon!_ if they've requested a removal.

**Comment by adamrusted** [comment]
On that thought, have we got a standardised and documented method for packages to auto-update?
Would make these potential improvements a _lot_ easier to make.

**Comment by mondeja** [comment]
> On that thought, have we got a standardised and documented method for packages to auto-update?

We have requested some third party packages to use a scheduled Github workflow comparing the new version of simple-icons with the current version of their package, but some people hasn't the necessary skills to make it work.

<details>

<summary>Example</summary>

The next `schedule` cron config means "At 00:00 on Monday, Tuesday, Wednesday, Thursday, and Friday".

```yaml
on:
  schedule:
    - cron: 0 0 * * 1,2,3,4,5
```

And then, something like...

```yaml
      - name: Compare release versions
        id: get-releases
        run: |
          simple_icons_version="$(curl --retry 5 -s https://api.github.com/repos/simple-icons/simple-icons/releases/latest | jq -r .tag_name)"
          echo "si=$simple_icons_version" >> $GITHUB_OUTPUT
          echo "lib=$(cat package.json | grep '"version":' | cut -d'"' -f4)" >> $GITHUB_OUTPUT
      - name: Bump version
        if: steps.get-releases.outputs.lib != steps.get-releases.outputs.si
```

</details>

**Comment by adamrusted** [comment]
Would it be worth us having a template repo with the actions and the NPM repo set up that others can use? We can then look to incorporate their webhook secret into the main project to trigger on a release - or have a separate 'release mgmt' project which abstracts those secrets away from the main repo.

**Comment by PeterShaggyNoble** [comment]
Has this been resolved by #9911 and the removal policy it introduces from v11 on?
